Jenkinsfile

```diff
@@ -32,6 +32,13 @@
  *****************************************************************************/
 
 node('linux_xenial_unprovisioned') {
+  def triggers = []
+  if (env.BRANCH_NAME == 'master') {
+    triggers << cron('H H(5-6) * * *')
```

Comment:
@jamiesnape why not just `H H * * *` or is the evening of the load not quite that smart?

Reply:
It needs to occur later than the nightly packages are built for Drake.

Reply:
Got it.